src/lang-promql/complete/promql.terms.ts

```diff
@@ -320,7 +320,16 @@ export const aggregateOpTerms = [
   },
 ];
 
-export const aggregateOpModifierTerms = [{ label: 'by' }, { label: 'without' }];
+export const aggregateOpModifierTerms = [
+  {
+    label: 'by',
+    info: 'Drop labels that are not listed in the "by" clause, even if their label values are identical between all elements of the vector',
+  },
+  {
+    label: 'without',
+    info: 'Remove the listed labels from the result vector, while all other labels are preserved the output.',
```

Comment:
```suggestion
    info: 'Remove the listed labels from the result vector, while all other labels are preserved in the output.',
```

Although I would maybe even keep it simpler:

```suggestion
    info: 'Remove the listed labels, preserve all others.',
```